src/docs/asciidoc/70-runtime.adoc

```diff
@@ -58,7 +58,7 @@ ComponentB --> ComponentA
 
 ....
 
-While developers could of course define the execution order via Spring's standard `@Order` annotation or `Ordered` interface, Spring Modulith provides an `ApplicationModuleInitializer` interface for beans to be run on application startup.
```

Comment:
Please scratch that. `@DependsOn` is not a recommended means for ordering user-facing components.

Reply:
I reverted the wording but I have a question: `@Order` is mentioned in the context of Application Module Initializers which are intended to be used for different modules, not multiple implementations of the same module:
> If a module B depends on module A, the initialization code of A has to run before the one for B, even if the initializers do not directly depend on another.

This is why I think the `@Order` parallel is confusing as it's intended to order multiple implementations of the same interface while initializers are intended for different modules altogether. Does this make sense?

Reply:
I think what plays into this is the fact that for module-specific initialization, you usually want transactional behavior. That's not something you can get with simple components and an `@PostConstruct` callback in components you'd interconnect via `@DependsOn`.

In other words, you typically declare an interface for the initializer with a custom method that can be `@Transactional` and a central component that would depend on `List<MyInitializer>` and invoke those transactional methods. For that list to be ordered, you'd either use `@Order`/`Ordered`. Pretty similar to what we actually implement with `ApplicationModuleInitializer`.

Reply:
@odrotbohm  Got it, thank you for the explanation!